pkg/config/lvmd/types.go

```diff
@@ -0,0 +1,52 @@
+// DeviceClass type definition and validation is taken from:
+// https://github.com/red-hat-storage/topolvm/blob/release-4.10/lvmd/device_class_manager.go
```

Comment:
Nit: We could add an empty line between doc and `package`. Otherwise godoc will treat the comment like doc for the package

Reply:
```suggestion
// https://github.com/red-hat-storage/topolvm/blob/release-4.10/lvmd/device_class_manager.go

```